simapp/simd/cmd/root.go

```diff
@@ -81,7 +80,8 @@ func initRootCmd(rootCmd *cobra.Command, encodingConfig params.EncodingConfig) {
 		debug.Cmd(),
 	)
 
-	server.AddCommands(rootCmd, simapp.DefaultNodeHome, newApp, createSimappAndExport, addModuleInitFlags)
+	a := appCreator{encodingConfig}
```

Comment:
```suggestion
	a := appCreator{encodingConfig}
```

Ouch, what a bad name for a variable... Mind renaming it to something more meaningful please?

Reply:
you mean `a`? It has a meaningful type name, and the object is only used in the line below, so a perfect candidate for a short name.

Reply:
BTW: this is very cool guidance for naming in Go: https://talks.golang.org/2014/names.slide#1